src/Components/CreateImageWizard/steps/Users/component/Empty.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {
+  Button,
+  EmptyState,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+  EmptyStateVariant,
+} from '@patternfly/react-core';
+import UserIcon from '@patternfly/react-icons/dist/esm/icons/user-icon';
+
+const Empty = () => {
```

Comment:
Maybe `EmptyUserState` would be a bit more descriptive? `Empty` is fairly generic.